src/components/button/index.stories.mdx

```diff
@@ -3,7 +3,7 @@ import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
 import Button from './index.tsx';
 import { Stack } from '../../index.ts';
 
-<Meta title="Components/Buttons/Button" component={Button} />
+<Meta title="Components/Button" component={Button} />
```

Comment:
Does it make senes moving the button into the forms category? or are we fine like this ay?

Reply:
I'd say it's fine. Only the submit variant is supposed to be used in forms. Most buttons will be outside of forms.